src/main/java/core/basesyntax/operation/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationHandler {
+    void getNewValueDependingOnOperation(FruitTransaction fruitTransaction);
```

Comment:
void method shouldn't contain 'get' in naming as it returns nothing
```suggestion
    void applyTransactionToStorage(FruitTransaction fruitTransaction);
```